plugins/modules/system/pam_limits.py

```diff
@@ -152,7 +158,7 @@ def main():
             use_max=dict(default=False, type='bool'),
             use_min=dict(default=False, type='bool'),
             backup=dict(default=False, type='bool'),
-            dest=dict(default=limits_conf, type='str'),
+            dest=dict(default=limits_conf, type='path'),
```

Comment:
Also needs a changelog fragments, as above (these should be `minor_changes` as this is a new feature).

Reply:
```suggestion
            dest=dict(default=limits_conf, type='str'),
```